FlaxEditor/Surface/VisjectSurface.Input.cs

```diff
@@ -284,10 +284,11 @@ public override bool OnMouseUp(Vector2 location, MouseButton buttons)
             SurfaceControl controlUnderMouse = GetControlUnderMouse();
 
             // Right clicking while attempting to connect a node to something
-            if (!_rightMouseDown && buttons == MouseButton.Right && !_isMovingSelection && _startBox != null)
+            if (!_rightMouseDown && !_isMovingSelection && _startBox != null)
```

Comment:
This block that shows the context menu should be moved down, after the:
```cs
if (base.OnMouseUp(location, buttons))
    return true;
```
that handles connecting boxes so if user stops pressing left mouse button over the other box they are connected without popup showing.